spec/lib/git_server/pull_request_spec.rb

```diff
@@ -69,25 +109,57 @@
   describe "#display_pull_request_summary" do
     subject            { GitReflow::GitServer::GitHub::PullRequest.new(pull_request).display_pull_request_summary }
 
-    before do
-      FakeGitHub.new(
-        repo_owner:   'reenhanced',
-        repo_name:    'repo',
-        pull_request: {
-          number:   pull_request.number,
-          owner:    pull_request.head.user.login,
-          comments: [{author: 'tito', body: 'lgtm'}, {author: 'ringo', body: ':+1:'}]
-        })
-      allow_any_instance_of(GitReflow::GitServer::GitHub::PullRequest).to receive(:build).and_return(Struct.new(:state, :description, :url).new)
-      GitReflow.git_server.stub(:find_open_pull_request).with({from: 'new-external-feature', to: 'master'}).and_return(pull_request)
+    context "Testing Pull Request Properties" do
+      before do
+        FakeGitHub.new(
+          repo_owner:   'reenhanced',
+          repo_name:    'repo',
+          pull_request: {
+            number:   pull_request.number,
+            owner:    pull_request.head.user.login,
+            comments: [{author: 'tito', body: 'lgtm'}, {author: 'ringo', body: ':+1:'}]
+          })
+        allow_any_instance_of(GitReflow::GitServer::GitHub::PullRequest).to receive(:build).and_return(Struct.new(:state, :description, :url).new)
+        GitReflow.git_server.stub(:find_open_pull_request).with({from: 'new-external-feature', to: 'master'}).and_return(pull_request)
+      end
+
+      it "displays relavent information about the pull request" do
+        expect{ subject }.to have_output("branches: new-external-feature -> reenhanced:master")
+        expect{ subject }.to have_output("number: #{pull_request.number}")
+        expect{ subject }.to have_output("url: #{pull_request.html_url}")
+        expect{ subject }.to have_output("reviewed by: #{"tito".colorize(:green)}, #{"ringo".colorize(:green)}")
+        expect{ subject }.to have_output("Last comment: \":+1:\"")
+      end
     end
 
-    it "displays relavent information about the pull request" do
-      expect{ subject }.to have_output("branches: new-external-feature -> reenhanced:master")
-      expect{ subject }.to have_output("number: #{pull_request.number}")
-      expect{ subject }.to have_output("url: #{pull_request.html_url}")
-      expect{ subject }.to have_output("reviewed by: #{"tito".colorize(:green)}, #{"ringo".colorize(:green)}")
-      expect{ subject }.to have_output("Last comment: \":+1:\"")
+    context "Testing Different LGTM Regex Expressions (Found Bug in :\\+1)" do
```

Comment:
` (Found Bug in :\\+1)` doesn't need to be in the context.  If explanation is needed on the default regex I think it belongs above the constant declaration where the regex is set.